### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by ritikarawat220**
### Tests for controllers  ✅

- Add spec for user controller
- Add spec for post controller

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -2,6 +2,16 @@
...
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns http success' do
+      get '/users/:user_id/posts/:id'
+      expect(response).to have_http_status(:success)
+    end
   end
```
Comment (danmainah) [inline spec/requests/posts_spec.rb:16]: - [ ] @ritikarawat220 great work adding test for the index and show actions, however you forgot to add tests to show `a correct template was rendered.` and `the response body includes correct placeholder text` for both index and show actions, kindly include the tests, they are part of project requirements as shown below
image

File: spec/requests/users_spec.rb [hunk 2]
```diff
@@ -2,6 +2,16 @@
...
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns http success' do
+      get '/users/show'
+      expect(response).to have_http_status(:success)
+    end
   end
```
Comment (danmainah) [inline spec/requests/users_spec.rb:16]: - [ ] @ritikarawat220 great work adding test for the index and show actions, however you forgot to add tests to show `a correct template was rendered.` and `the response body includes correct placeholder text` for both index and show actions, kindly include the tests, they are part of project requirements as shown below
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @ritikarawat220 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️  Grouped the tests correctly and used RSpec for testing
✔️  Ensured all tests added are passing


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @ritikarawat220 , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed
- [ ] all test passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**